pkg/goDB/info/hostid_linux.go

```diff
@@ -0,0 +1,34 @@
+//go:build linux
+// +build linux
+
+package info
+
+import (
+	"errors"
+	"io/fs"
+	"os"
+)
+
+const (
+	machineIDPath     = "/etc/machine-id"
+	machineIDDBusPath = "/var/lib/dbus/machine-id"
+)
+
+func hostID() (string, error) {
+
+	// Attempt to read the machine ID from the main file
+	idData, err := os.ReadFile(machineIDPath)
+	if err != nil {
+		if !errors.Is(err, fs.ErrNotExist) {
+			return UnknownID, err
```

Comment:
I'm wondering if it still makes sense to try the next case as well and only then fail.

Reply:
Ah, you mean try the second file on _any_ error, not just on `file does not exist`. Yeah, why not, can't really do any harm...